website/docs/basics/blocks.md

```diff
@@ -3,3 +3,50 @@ id: blocks
 title: Blocks
 sidebar_label: Blocks
 ---
+
+Since WordPress 5.0, we were introduced to the new editing experience using block. With blocks, the idea is to have the same look and feel in the editor as it is on the frontend. As a developer, you just got a new technology that you should know if you want to do a great project on WordPress. That new technology is React because everything regarding blocks is done using React. 
+
+Don't get discouraged just yet, because the WordPress team has made the process of creating blocks so easy. But we are not here to talk about the simple stuff. We want to do more complex projects with custom blocks, so follow along and see what we have prepared here.
```

Comment:
```suggestion
Don't get discouraged just yet, because the WordPress team has made the process of creating blocks relatively easy. But we are not here to talk about the simple stuff. We want to do more complex projects with custom blocks, so follow along and see what we have prepared here.
```

Well, I don't think the WP community would agree with the `the WordPress team has made the process of creating blocks relatively easy` part 😂